css/css-counter-styles/ethiopic-numeric/css3-counter-styles-068.html

```diff
@@ -18,18 +18,19 @@
 <p class="instructions">Test passes if the two columns are the same, ignoring the suffix.</p>
 
 
-<div class='test'><ol>
```

Comment:
was this change intended?

Reply:
Yes, the style of the tests in this directory was so weird I couldn't make sense of it without adding a few newlines.